### PR TITLE
27 Jan

### PR DESCRIPTION
**Pull request by Shivam8905**

### CODE REVIEW

File: Squres.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+num1 = int(input("Enter start range:"))
+num2 = int(input("Enter end range:"))
+
+number = [*range(num1, num2, 1)]
```
Comment (Jdpatel32) [inline Squres.py]: range(num1, num2 +1)

File: Squres.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+square = [value ** 2 for value in range(num1, num2)]
```
Comment (Jdpatel32) [inline Squres.py]: [value ** 2 for value in number]

File: Squres.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+database = dict(zip(number, square))
+
+counter=0
+mySet = set()
+
+while  counter<3:
```
Comment (Jdpatel32) [inline Squres.py:12]: what if counter is greater than 3?

Comment (Shivam8905) [reply Squres.py:12]: if the counter is greater than 3 means you have tried the wrong input 3 times so the program will be terminated

Comment (Jdpatel32) [reply Squres.py:12]: ok but how user know ?

**Review by Jdpatel32 — APPROVED** [review]

**Pull request merged**